app/views/admin/edition_images/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :context, "#{@edition.title}" %>
 <% content_for :page_title, "Images for #{@edition.format_name}" %>
 <% content_for :title, "Images for #{@edition.format_name}" %>
-<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @new_image, parent_class: "edition_images")) if @new_image.present? %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @new_image.image_data, parent_class: "edition_images")) if @new_image.present? %>
```

Comment:
Looks like we'd lose any validation issues on the @new_image object with this change, I can see that still has validation rules.